corehq/pillows/domain.py

```diff
@@ -61,3 +61,20 @@ def change_transform(self, doc_dict):
         for country in countries:
             doc_ret['deployment']['countries'].append(COUNTRIES[country].upper())
         return doc_ret
+
+
+def get_domain_reindexer():
+    pillow = DomainPillow(online=False)
+    return ElasticPillowReindexer(
+        pillow=pillow,
+        change_provider=CouchViewChangeProvider(
+            document_class=Domain,
+            view_name='all_docs/by_doc_type',
+            view_kwargs={
+                'startkey': ['Domain'],
+                'endkey': ['Domain', {}],
+            }
+        ),
+        elasticsearch=pillow.get_es_new(),
```

Comment:
good point - was just mimicking what we had done elsewhere and wondering if we needed to plan for a world where we have multiple clients based on index type like we do with couchdbs. do we think we'll always only have one ES client? can't think of why that would ever need to change.